bogota-july-2018/todo-react-list/src/Components/TodoList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ListElement from './ListElement.js';
+
+export default class TodoList extends React.Component {
```

Comment:
I know this is valid and as far as I know is not a bad practice. 

But as a personal advise, I would like you to suggest that you can write your component like: 

```
class Component extends React.component {
    // your code
}

export default Component
```

That will allow you to put easy use HOC as connect

```
export default connect(mapStateToProps, mapDispatchToProps)(Component)
```